mlir/include/mlir/Dialect/SPIRV/IR/SPIRVMatrixOps.td

```diff
@@ -198,4 +197,53 @@ def SPIRV_TransposeOp : SPIRV_Op<"Transpose", [Pure]> {
 
 // -----
 
+def SPIRV_VectorTimesMatrixOp : SPIRV_Op<"VectorTimesMatrix", [
+    Pure,
+    AllElementTypesMatch<["vector", "result"]>
```

Comment:
Could you also update `MatrixTimesVector` above to use the same type constraints?